types/types.go

```diff
@@ -170,7 +170,8 @@ const (
 
 	BackingImageDataSourcePodNamePrefix = "backing-image-ds-"
 
-	shareManagerPrefix    = "share-manager-"
+	shareManagerPrefix = "share-manager-"
+
```

Comment:
NIT: no need for a separate section for this variable.

Reply:
Incorporated suggestions!